maitake/Cargo.toml

```diff
@@ -17,7 +17,7 @@ categories = [
     "async",
 ]
 edition = "2021"
-rust-version = "1.61.0"
+rust-version = "1.76.0"
```

Comment:
again, i think the MSRV bump isn't strictly necessary unless we are also changing the cordyceps dep